src/main/java/io/github/townyadvanced/townyprovinces/settings/ConfigNodes.java

```diff
@@ -97,7 +97,7 @@ public enum ConfigNodes {
 		"# This value determines the weight of the border."),
 	CIVILIZED_BORDER_APPEARANCE_OPACITY(
 		"province_types.civilized.border_appearance.opacity",
-		"0.9",
+		"1",
```

Comment:
🚧 
- Please revert to 0.9
- This is to ensure the wasteland colour "wins" more often at the border